test/apps/iodemo/run_io_demo.sh

```diff
@@ -132,7 +132,7 @@ usage()
 	echo "  -h|--help                   Show this help message"
 	echo "  -v|--verbose                Turn on verbosity"
 	echo "  -H|--hostlist <h1>,<h2>,..  List of host names to run on"$(show_default_value host_list)
-	echo "  -i|--netif <n>              Network interface to use"$(show_default_value net_if)
+	echo "  -i|--netif <n>              Network interfaces to use"$(show_default_value net_ifs)
```

Comment:
`echo "  -i|--netif <n1,n2>              Comma-separated list of network interfaces to use"$(show_default_value net_ifs)`